Project_Starting_Point/Client/Client.cpp

```diff
@@ -21,39 +21,41 @@ int main()
 	ClientSocket = socket(AF_INET, SOCK_STREAM, IPPROTO_TCP);
 	SvrAddr.sin_family = AF_INET;
 	SvrAddr.sin_port = htons(27001);
-	SvrAddr.sin_addr.s_addr = inet_addr("127.0.0.1");
-	connect(ClientSocket, (struct sockaddr*)&SvrAddr, sizeof(SvrAddr));
+	SvrAddr.sin_addr.s_addr = inet_addr("127.0.0.1"); 
+	connect(ClientSocket, (struct sockaddr*)&SvrAddr, sizeof(SvrAddr)); // connecting client with server
 
-	uiSize = GetSize();
-	for (unsigned int l = 0; l < uiSize; l++)
+	uiSize = GetSize(); // calling in-home function which returns size O(n)
+	for (unsigned int l = 0; l < uiSize; l++)  //O(n)
```

Comment:
This loop handles reading from disk and sending data to the server. We should note that here.